runtime/compiler/maven.vim

```diff
@@ -0,0 +1,43 @@
+" Vim compiler file
+" Compiler:	Maven
+" Maintainer:	D. Ben Knoble <ben.knoble+vim@gmail.com>
+" Maintainer:	Konfekt
+" Original Source:	https://github.com/JalaiAmitahl/maven-compiler.vim/blob/master/compiler/mvn.vim (Copyright Dan Taylor, distributed under the same terms as LICENSE)
+" Original Source:	https://github.com/mikelue/vim-maven-plugin/blob/master/compiler/maven.vim (distributed under same terms as LICENSE per https://github.com/mikelue/vim-maven-plugin/issues/13)
+" Last Change:	2024 Nov 12
+
+if exists("current_compiler")
+  finish
+endif
+let current_compiler = "maven"
+
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
We don't necessarily need the `s:cpo_save` dance here I think

Reply:
Ah, probably not. I copied boilerplate from nearby and didn't give it much thought. Fixed in a6048d4f00 (runtime/compiler: add a Maven compiler plugin, 2024-11-12).

<details><summary>range-diff:</summary>

```
1:  368056b0f ! 1:  a6048d4f0 runtime/compiler: add a Maven compiler plugin
    @@ runtime/compiler/maven.vim (new)
     +endif
     +let current_compiler = "maven"
     +
    -+let s:cpo_save = &cpo
    -+set cpo&vim
    -+
     +CompilerSet makeprg=mvn\ --batch-mode
     +
     +" Error message for POM
    @@ runtime/compiler/maven.vim (new)
     +" Misc message removal
     +CompilerSet errorformat+=%-G[INFO]\ %.%#,
     +CompilerSet errorformat+=%-G[debug]\ %.%#
    -+
    -+let &cpo = s:cpo_save
    -+unlet s:cpo_save
```

</details>